src/dao/teamDao.js

```diff
@@ -0,0 +1,22 @@
+var teamDb = require('../model/teamModel');
+var mongoose = require("mongoose");
+class TeamDao {
+    constructor() {}
+
+    /*Return team by id*/
+    getTeamById(teamId) {
+        const getTeamByIdPromise = new Promise((resolve, reject) => {
+            var objectIdTeamId = mongoose.mongo.ObjectID(teamId);
+            teamDb.findOne({'_id':objectIdTeamId}, function (err, teamFound) {
+                if (err) {
+                    reject(err);
+                } else {
+                    resolve(teamFound);
+                }
+            });
+        });
+        return getTeamByIdPromise;
+    };
+}
+
+module.exports = TeamDao;
```

Comment:
Add EOF and also a Eslint rule to prevent this, if eslint is missing in this project add it.

Reply:
eslint added and error were fixed